app/src/main/java/com/tehmou/rxbookapp/view/RepositoryView.java

```diff
@@ -53,5 +62,20 @@ private void setRepository(@NonNull GitHubRepository repository) {
         titleTextView.setText(repository.getName());
         stargazersTextView.setText("stars: " + repository.getStargazersCount());
         forksTextView.setText("forks: " + repository.getForksCount());
+        request = Glide.with(getContext())
```

Comment:
@jaggernod probably we should release the request if it is already ongoing. Nothing prevents from setting the view model twice.